multicluster/cmd/allow.go

```diff
@@ -69,16 +74,40 @@ func newAllowCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			stdout.Write(buf.Bytes())
-			stdout.Write([]byte("---\n"))
 
-			return nil
+			if opts.output == "json" {
```

Comment:
Thought / question, would we be able to use `pkgcmd.RenderYAMLs` here?

```
pkgcmd.RenderYAMLs(&buf, os.Stdout, opts.output)
```

or would it not work? sometimes extensions have their own little logic embedded in the commands and I'm wondering if that's the case here.